fleetshard/pkg/central/postgres/dbconnection.go

```diff
@@ -14,7 +16,22 @@ type DBConnection struct {
 	password string
 }
 
-const sslMode = "require"
+var once sync.Once
+var rdsCertificateData []byte
+
+const (
+	// CentralRDSCertificateBaseName is the name of the additional CA that is passed to Central
+	CentralRDSCertificateBaseName = "rds-ca-bundle"
+
+	sslMode = "verify-full"
+
+	caPath = "/usr/local/share/ca-certificates/"
+
+	rdsCertificatePath = caPath + "aws-rds-ca-global-bundle.pem"
```

Comment:
Why so many spaces between the constant definitions?

Reply:
Can you add information that this is used to trust AWS RDS instances? (not authentication)

```suggestion
	rdsCertificatePath = caPath + "aws-rds-ca-global-bundle.pem"
```

Reply:
Done